org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -70,7 +70,7 @@ GithubOidcSageBionetworksDataCuratorInfra:
   TemplatingContext:
     Repositories:
       - name: "data_curator-infra"
-        branch: "main"
+        branch: "*"
```

Comment:
What do you think of splitting this into two entries, one to deploy from `dev` branch into `DnTDevAccount` and another to deploy from `prod` branch into `DCAProdAccount`? That would help prevent an application developer from accidentally deploying dev' changes to production.